src/pages/pupils/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug]/index.tsx

```diff
@@ -0,0 +1,89 @@
+import {
+  GetStaticPathsResult,
+  GetStaticProps,
+  GetStaticPropsResult,
+  NextPage,
+} from "next";
+import { oakDefaultTheme, OakThemeProvider } from "@oak-academy/oak-components";
+
+import { QuizEngineProvider } from "@/components/PupilJourneyComponents/QuizEngineProvider/QuizEngineProvider";
+import { QuizRenderer } from "@/components/PupilJourneyComponents/QuizRenderer/QuizRenderer";
+import getPageProps from "@/node-lib/getPageProps";
+import curriculumApi2023 from "@/node-lib/curriculum-api-2023";
+import { PupilLessonOverviewData } from "@/node-lib/curriculum-api";
+import {
+  getFallbackBlockingConfig,
+  shouldSkipInitialBuild,
+} from "@/node-lib/isr";
+
+export type PupilLessonOverviewPageProps = {
+  curriculumData: PupilLessonOverviewData;
+};
+
+const Pupils: NextPage<PupilLessonOverviewPageProps> = ({ curriculumData }) => {
+  const { starterQuiz } = curriculumData;
+
+  return (
+    <OakThemeProvider theme={oakDefaultTheme}>
+      <QuizEngineProvider questionsArray={starterQuiz ?? []}>
+        <QuizRenderer />
+      </QuizEngineProvider>
+    </OakThemeProvider>
+  );
+};
+
+export type URLParams = {
```

Comment:
URLParams appears in all the pages, but lets not blindly copy patterns. It would be nicer if this had a unique name to disambiguate eg. PupilPageURLParams.